subhub-link-checker/README.md

```diff
@@ -12,6 +12,21 @@ Written with React/TypeScript/Contentful App Framework.
 Part of the Hub Expansion project. This is required due to SubHub routing logic in the frontend.
 
 There are three instances of the Contentful App - one for dev, test and prod. [Read more...](https://www.contentful.com/developers/docs/extensibility/app-framework/)
+
+## Seeing and deploying your changes
+This assumes the Contentful environments are set up with the required app definitions. See the First time deployment section if you need to redo those steps.
+
+ResearchHub has three Contentful environments - `dev`, `test`, and `prod`, and `prod` is the one content authors see. Use `dev` for previewing your changes and manual testing - Contentful Apps can't be run locally. To deploy to `test` and `prod`, you should perform the deployment during a scheduled Change.
+
+1. If you are deploying to `test` and `prod`, first check-in your changes to Git and merge them into `master`.
```

Comment:
Maybe mention here that deployment to test and prod should only be done as part of the official release process (see: https://wiki.auckland.ac.nz/display/APPLCTN/Release+to+prod) and that content authors need to be notified before deploying updates.

Reply:
Thanks Rose, I've added your writeup into the README!